### PR TITLE
Add low light boost settings, and UI indicator

### PR DESCRIPTION
**Pull request by yasith**
UI Indicator:
Screenshot 2024-05-30 at 2 01 49 PM

Quick settings:
Screenshot 2024-05-30 at 2 01 17 PM

Screenshot 2024-05-30 at 2 00 47 PM

Comments:
Disabling flash when low light boost is turned on, is handled by a PR down the line.

### CODE REVIEW

**Review by Kimblebee — COMMENTED** [review]
code looks good :)

just a few small comments to address

File: feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt [hunk 1]
```diff
@@ -336,6 +339,22 @@ fun StabilizationIcon(
     }
 }
 
+@Composable
+fun LowLightBoostIcon(lowLightBoost: LowLightBoost, modifier: Modifier = Modifier) {
+    when (lowLightBoost) {
+        LowLightBoost.ENABLED -> {
+            Icon(
+                imageVector = Icons.Outlined.Nightlight,
```
[the remark this answers is not in the document]
Comment (yasith) [reply feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt:347]: This icon changes to the filled state, when low light boost is enabled and active.

Icon behavior:
- disabled: hidden
- enabled and inactive: outline
- enabled and active: filled

[the remark this answers is not in the document]
Comment (Kimblebee) [reply feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt:347]: oh ok. I think you should include a comment stating that behavior.

I'd also suggest to lower the alpha for the "enabled and inactive" case to reduce its priority on the screen

**Review by Kimblebee — APPROVED** [review]

**Comment by Jaehwa-Noh** [comment]
It looks like just adding the indicator not actual functions, am I missing anything?

**Comment by yasith** [comment]
> It looks like just adding the indicator not actual functions, am I missing anything?

Yes, this is the first of a chain of PRs

**Pull request merged**